modules/apps/transfer/keeper/grpc_query.go

```diff
@@ -156,3 +159,17 @@ func (k Keeper) TotalEscrowForDenom(c context.Context, req *types.QueryTotalEscr
 		Amount: amount,
 	}, nil
 }
+
+// DenomTrace implements the Query/DenomTrace gRPC method
+// Deprecated: Please use Denom instead.
+func (Keeper) DenomTrace(c context.Context, req *types.QueryDenomTraceRequest) (*types.QueryDenomTraceResponse, error) {
+	// TODO: Return sensible error.
+	return nil, nil
+}
+
+// DenomTraces implements the Query/DenomTraces gRPC method
+// Deprecated: Please use Denoms instead.
+func (Keeper) DenomTraces(c context.Context, req *types.QueryDenomTracesRequest) (*types.QueryDenomTracesResponse, error) {
+	// TODO: Return sensible error.
```

Comment:
we use `ErrUnsupported` in ICA, and `ErrUnsupportedAction` in fee, maybe we can use something similar